syz-manager/manager.go

```diff
@@ -1084,6 +1084,12 @@ func (mgr *Manager) saveRepro(res *ReproResult) {
 	if len(cprogText) > 0 {
 		osutil.WriteFile(filepath.Join(dir, "repro.cprog"), cprogText)
 	}
+	for idx, mount := range repro.Prog.MountedImages() {
+		if mount.Reader != nil {
+			osutil.WriteFileStream(filepath.Join(dir, fmt.Sprintf("repro.mount%d", idx)),
```

Comment:
I think we need to save them in some compressed format. Each of them can be ~100MB.